doc/classes/AudioServer.xml

```diff
@@ -132,7 +132,7 @@
 		<method name="get_output_latency" qualifiers="const">
 			<return type="float" />
 			<description>
-				Returns the audio driver's effective output latency. This is based on [member ProjectSettings.audio/driver/output_latency], but the exact returned value will differ depending on the operating system and audio driver.
+				Returns the audio driver's effective output latency (in seconds). This is based on [member ProjectSettings.audio/driver/output_latency], but the exact returned value will differ depending on the operating system and audio driver.
```

Comment:
I think it's important to clarify, since in settings the latency is in milliseconds, and it was not very clear for me what actually was returned

Reply:
Thumbs up, I don't think the parentheses are needed though

Reply:
Put parentheses just to match Godot's style? However, when I look at all the times "in seconds" is used it doesn't seem to be very consistent, sometimes it's written in parentheses, with comma or without anything, e.g. `Should return the length of this audio stream, in seconds` and `Returns the length of the audio stream in seconds` in AudioStream's doc. Will push just `Returns the audio driver's effective output latency in seconds`

Reply:
There's no consistency to it. It depends on the rest of the sentence. I think in this case there's so few things to read, there's no need for parentheses. "_Returns the audio driver's effective output latency in seconds_" sounds good to me.

Reply:
Pushed the change, so will resolve this conversation